doc/fd.1

```diff
@@ -455,5 +455,7 @@ $ fd -e py
 .TP
 .RI "Open all search results with vim:"
 $ fd pattern -X vim
+.SH BUGS
+Bugs can be reported on Github: https://github.com/sharkdp/fd
```

Comment:
If we're talking about bugs, should we link directly to https://github.com/sharkdp/fd/issues?